<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accueil</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_planification">
         <attribute name="title">
          <string>Planification</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_Client">
         <attribute name="title">
          <string>Client</string>
         </attribute>
         <widget class="QLabel" name="lab_NomRecherche">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>260</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_NomRecherche">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>260</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_PrenomRecherche">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>290</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="lab_PrenomRecherche">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>290</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Pr√©nom :</string>
          </property>
         </widget>
         <widget class="QLabel" name="lab_DateMiniRecherche">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>320</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>De :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_IdRecherche">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>230</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Id :</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_IdRecherche">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>230</y>
            <width>131</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>591</width>
            <height>194</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTableView" name="tableViewClient">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_Recherche">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>210</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Recherche :</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_ModifierClient">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>230</y>
            <width>251</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_SupprimerClient">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>310</y>
            <width>251</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEditMini">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>320</y>
            <width>131</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximumDate">
           <date>
            <year>2099</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>1999</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEditMax">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>350</y>
            <width>131</width>
            <height>22</height>
           </rect>
          </property>
          <property name="maximumDate">
           <date>
            <year>2099</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>1999</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
         <widget class="QLabel" name="lab_DateMaxRecherche">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>350</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>A :</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_Personnel">
         <attribute name="title">
          <string>Personnel</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter</string>
     </property>
     <addaction name="actionClient"/>
     <addaction name="actionPersonnel"/>
    </widget>
    <addaction name="menuAjouter"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_client_Tool"/>
   <addaction name="action_Personnel_Tool"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="actionPersonnel">
   <property name="text">
    <string>Personnel</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="action_client_Tool">
   <property name="icon">
    <iconset>
     <normaloff>../ressource/clients.png</normaloff>../ressource/clients.png</iconset>
   </property>
   <property name="text">
    <string>Client</string>
   </property>
   <property name="toolTip">
    <string>Ajouter un client</string>
   </property>
  </action>
  <action name="action_Personnel_Tool">
   <property name="icon">
    <iconset>
     <normaloff>../ressource/personnel.ico</normaloff>../ressource/personnel.ico</iconset>
   </property>
   <property name="text">
    <string>Personnel</string>
   </property>
   <property name="toolTip">
    <string>Ajouter un personnel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
